weave/trace_server/clickhouse_trace_server_batched.py

```diff
@@ -770,6 +770,12 @@ def add_new_row_needed_to_insert(row_data: Any) -> str:
         return tsi.TableUpdateRes(digest=digest, updated_row_digests=updated_digests)
 
     def table_query(self, req: tsi.TableQueryReq) -> tsi.TableQueryRes:
+        rows = list(self.table_query_stream(req))
```

Comment:
By using the stream method here, we effectively share all the same code and existing tests